Tests/SVT-HEVC_FunctionalTests.py

```diff
@@ -893,6 +961,18 @@ def run_validation_test(self, seq_list):
         num_tests, num_passed = self.me_hme_test(seq_list)
         total_tests = total_tests + num_tests
         total_passed = total_passed + num_passed
+        num_tests, num_passed = self.hdr_test(seq_list)
+        total_tests = total_tests + num_tests
```

Comment:
total_tests = total_tests + num_tests
        total_passed = total_passed + num_passed	      
Could you make these two lines a separate function?